### PR TITLE
2018 update #1

### PR DESCRIPTION
**Pull request by saschaben**

### CODE REVIEW

File: _posts/2018-01-23-updates.markdown [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: Happy New Year, From Team ilios!
+date: 2018-01-23  15:00:00
+categories: updates
+---
+
+We of the Ilios team welcome you to the new year, and the first release of 2018, __v3.42.0__, which adds further permissions improvements, additional data to downloaded reports, and a calendar view for learner groups -- aloing with a bunch of other improvements to start off the year right.
```
Comment (Trott) [inline _posts/2018-01-23-updates.markdown]: Oh, come on, seriously? "We of the Ilios team"? Are we the Stonecutters or something?

Regardless of how you feel about that: `along` is misspelled on this line.

Also, I would strongly recommend avoiding the ` -- `. It seems to be used incorrectly here.  And the sentence is a run-on sentence anyway, so removing the ` -- ` might help that too.

My counterproposal:

> Welcome to the New Year and the first release of 2018, __v3.42.0__. This release adds permissions improvements, enhancements to downloaded reports, a calendar view for learner groups, and a bunch of other improvements!

File: _posts/2018-01-23-updates.markdown [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+We are looking forward to another extremely active and productive year in the Ilios community and development team, and we hope that you will continue to find increasing value in the platform for your students, faculty, and administration.
+
+If you haven't yet had a chance to update your systems to the latest release, now is a great time to do so!
+
+Questions? Comments? Feedback? Please let us know at [support@iliosproject.org](mailto:support@iliosproject.org) or in [https://team-ilios.slack.com/messages/help/](https://team-ilios.slack.com/messages/help/) (if you have not yet joined our slack channel, you can get started here: [https://ilios-slack.herokuapp.com/](https://ilios-slack.herokuapp.com/)).
```
Comment (Trott) [inline _posts/2018-01-23-updates.markdown]: I know this is boilerplate, but I have some punctuation corrections/suggestions:

> Questions? Comments? Feedback? Please let us know at [support@iliosproject.org](mailto:support@iliosproject.org) or in [https://team-ilios.slack.com/messages/help/](https://team-ilios.slack.com/messages/help/).  (If you have not yet joined our slack channel, you can get started at [https://ilios-slack.herokuapp.com/](https://ilios-slack.herokuapp.com/).)

**Comment by Trott** [comment]
Updated formatting of all my comments so that my suggestions are now copy/paste-able directly into the markdown file.

File: _posts/2018-01-23-updates.markdown [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: Happy New Year, From Team ilios!
+date: 2018-01-23  15:00:00
+categories: updates
+---
+
+We of the Ilios team welcome you to the new year, and the first release of 2018, __v3.42.0__, which adds further permissions improvements, additional data to downloaded reports, and a calendar view for learner groups -- aloing with a bunch of other improvements to start off the year right.
+
+We are looking forward to another extremely active and productive year in the Ilios community and development team, and we hope that you will continue to find increasing value in the platform for your students, faculty, and administration.
```
[the remark this answers is not in the document]
Comment (saschaben) [reply _posts/2018-01-23-updates.markdown]: Leaving as is. Sounds less terse to my reading ear as a compound sentence.

File: _posts/2018-01-23-updates.markdown (hunk 2 above)
Comment (saschaben) [reply _posts/2018-01-23-updates.markdown]: updated. thx.

**Pull request merged**